### ISSUE TITLE
How do I upgrade from dev-master to 1.0?

### ISSUE THREAD

**Issue by sysmesh** [thedevdojo/voyager#1900]
- Laravel Version: 5.5.1
- Voyager Version: dev-master
- PHP Version: 7.1.9
- Database Driver & Version:

### Description:
There's no information on how to upgrade from the dev-master (used it for Laravel 5.5 as Voyager wasn't released at the time).
How do I upgrade from dev-master to 1?

**Comment by VictorAvelar** [comment]
There you go.

https://voyager.readme.io/docs/upgrading

**Comment by sysmesh** [comment]
Thanks but these instructions do not contain information on moving from the dev version to the release version. I did try them but none did the trick :-(

**Comment by VictorAvelar** [comment]
Can you describe the problem pls.

**Comment by sysmesh** [comment]
Seems I had a problem with my Git client. I fixed it and the upgrade worked but...
Even though I cleared cache and views there's a weird behavior: 
Some stuff simply is missing on the menu/bread. 

For example: in the menu builder I got menu items (for the admin) that appear on the builder just fine yet do not show on the actual menu at all (simply missing). I tried to refresh but they are still missing.

The weird part is that in the menu_items table I can see all items but in the actual admin side bar some are missing.

Another example is a recent Bread I added to a table isn't showing the edit/delete icons on each line (it did show them before moving from the dev version).

Any idea?
